src/main/scala/chisel3/util/experimental/BoringUtils.scala

```diff
@@ -224,6 +228,10 @@ object BoringUtils {
     def drill(source: A, path: Seq[BaseModule], connectionLocation: Seq[BaseModule], up: Boolean): A = {
       path.zip(connectionLocation).foldLeft(source) {
         case (rhs, (module, conLoc)) if (module.isFullyClosed) => boringError(module); DontCare.asInstanceOf[A]
+        case (rhs, (module, conLoc)) if (up && module == path(0) && isPort(rhs)) => {
```

Comment:
Nit: `conLoc` is unused:

```suggestion
        case (rhs, (module, _)) if (up && module == path(0) && isPort(rhs)) => {
```